evm/loomevm.go

```diff
@@ -209,12 +216,18 @@ func (lvm LoomVm) Call(caller, addr loom.Address, input []byte, value *loom.BigU
 			)
 		}
 
+		ethereumTxHash := types.NewTransaction(
+			uint64(auth.Nonce(lvm.state, caller)), common.BytesToAddress(addr.Local),
+			nil, math.MaxUint64, big.NewInt(0), input,
```

Comment:
Here too the `value` shouldn't be `nil`.

Reply:
Done